### DISCUSSION TITLE
Quarkus World Tour 2023 - Call for JUG Tour Stops

### DISCUSSION THREAD

**Discussion by insectengine** [quarkusio/quarkus#23453] [Events]
**Want to have a tour stop at your JUG?**

**Post a comment** to start the conversation and let’s get you on the list! 

Please provide the following information to get started:

1. Name of the JUG
2. Best JUG Contact information
3. Language preference
4. Country and timezone
5. Any specific Quarkus related content you’d be interested in hearing about during your session

Anything else - email [worldtour@quarkus.io](mailto:worldtour@quarkus.io)

**Comment by pedro-hos** [comment]
Hello, 

1. Name of the JUG: JUG Vale (https://github.com/Jug-Vale)
2. Best JUG Contact information: @jesuino (william.fatecsjc@gmail.com) and I (pedro-hos@outlook.com)
3. Language preference: English or Portuguese
4. Country and timezone: Brazil (GMT-3)
5. Any specific Quarkus related content you’d be interested in hearing about during your session: No, any content would be good.

Thank you very much

**Reply by arnaudbos** [reply]
Hi 👋, 


1. Name of the JUG
[ToulouseJUG](https://www.toulousejug.org/)

2. Best JUG Contact information
[jullienzan](https://twitter.com/jullienzan) and [myself](https://twitter.com/arnaud_bos)

3. Language preference
French or English

4. Country and timezone
France UTC+1

5. Any specific Quarkus related content you’d be interested in hearing about during your session
No limit!

**Reply by insectengine** [reply]
@pedro-hos, @arnaudbos  -  Great! We'll be reaching out to make this happen!

**Reply by tiago-marques** [reply]
Brazil of course ! Please make this possible !

**Reply by charger28** [reply]
@pedro-hos - just sent you a private email to solidify some more details.

**Reply by charger28** [reply]
@arnaudbos - just sent you an email to solidify some details.

**Reply by charger28** [reply]
@arnaudbos - send me an email (jebeck@redhat.com) when you get a chance. We would like to solidify a date and topics for the tour stop. Thanks.

**Reply by insectengine** [reply]
@arnaudbos  - I sent you an email (from jcobb@redhat.com) regarding a Toulouse stop.

**Comment by evrentan** [comment]
Hello @insectengine,
Pls find my request for Turkey Java Community;

1. Name of the JUG: Turkey Java Community (https://turkeyjava.com)
2. Best JUG Contact information: info@turkeyjava.com, and @evrentan 
3. Language preference: English
4. Country and timezone: Turkey (GMT+3)
5. Any specific Quarkus related content you’d be interested in hearing about during your session: Any topic will be okay, there is no limit :)

Thanks,
Evren

**Reply by charger28** [reply]
Thanks for reaching out Evren. We will be in touch with you soon to schedule a tour stop.

**Reply by charger28** [reply]
@evrentan - just sent you an email to schedule the details.

**Reply by evrentan** [reply]
Hi @charger28, thanks for your email. Just answered it :)

**Comment by anthonydahanne** [comment]
Hello Quarkus Team,

1. Name of the JUG

MontrealJUG

2. Best JUG Contact information

info@montrealjug.org

3. Language preference

French preferred, English OK

4. Country and timezone

Canada, Province of Québec Eastern Timezone

5. Any specific Quarkus related content you’d be interested in hearing about during your session

We already presented great Quarkus content in the past: Workshop [Quarkus, par le Montréal JUG ](https://www.montreal-jug.org/meetup/workshop-quarkus/), [Midi Quarkus - Emmanuel Bernard et Antonio Goncalves ](https://www.montreal-jug.org/meetup/midi-quarkus/) - but any content on the roadmap, next features would be appreciated - if possible in person.

Thank you!

**Reply by charger28** [reply]
Thanks for reaching out Anthony. We will be in touch with you soon to schedule a tour stop.

**Reply by charger28** [reply]
@anthonydahanne - we just sent and email to info@maontrealjug.org to schedule the stop and solidify the details.

**Reply by anthonydahanne** [reply]
Hello @charger28 - thanks for sending your initial email to the MontrealJUG on March 4th and answer on March 31st.
Since then, I've replied to you on March 31st providing answers to all your questions, then I bumped the message on April 5th, but to this day, still no answer.

Is the Quarkus World JUG tour still on? Is it cancelled? Why don't you answer our emails to finalize an event?
thank you

**Reply by anthonydahanne** [reply]
hello @charger28 - thanks for getting back to us. Yes, let's do June, gonna be awesome!
Thanks!

**Reply by jjpinto** [reply]
Which day? Will it be in French or English? @anthonydahanne

**Comment by craig-mcd** [comment]
Hello Quarkus,

1. Manchester Java Community (https://www.meetup.com/ManchesterUK-Java-Community)
2. organisers@manchesterjavacommunity.org
3. English
4. UK, UTC 0 or UTC +1
5. Anything except an intro talk as we had one with from the 2021 tour with Georgios Andrianakis.

Thanks!

**Reply by charger28** [reply]
Thanks for reaching out Craig. We will be in touch soon to schedule your tour stop.

**Reply by charger28** [reply]
@CraigMcDonaldCodes - just sent an email to solidify the date and topics.

**Comment by tgallin** [comment]
Hi Quarkus Team,

We would love to have a stop at Orléans

   1. Name of the JUG : 
      [JUG Orléans](https://www.jugorleans.fr/)

   2. Best JUG Contact information
      [myself](https://twitter.com/GallinThomas) or [jugorleans@gmail.com](jugorleans@gmail.com)

   3. Language preference
      French or English

   4. Country and timezone
      France (UTC+1)

   5. Any specific Quarkus related content you’d be interested in hearing about during your session
      Any content would be great

**Reply by charger28** [reply]
Thanks for reaching out Thomas. We will be in touch soon to schedule your tour stop.

**Reply by charger28** [reply]
@tgallin - just sent an email to solidify dates/topics.

**Comment by anyulled** [comment]
**Name of the JUG:** 
BarcelonaJug https://www.meetup.com/es-ES/BarcelonaJUG/

**Best JUG Contact information:** 
@anyulled- anyulled@barcelonajug.org

**Language preference:** 
English, Spanish, Catalan

**Country and timezone:** 
Spain UTC +1

**Any specific Quarkus related content you’d be interested in hearing about during your session:** 
Last year we had Roberto Cortez having the session and it went pretty great, we would like to explore the capabilities of interaction with cloud providers, and serverless, like using the aws-lambda extension

**Reply by charger28** [reply]
Thanks for reaching out Anyul. We will follow up with you privately to schedule and details.

**Comment by xstefank** [comment]
Hi,

Name of the JUG: CZJUG
Best JUG Contact information: @xstefank xstefank122@gmail.com
Language preference: Czech, Slovak, English
Country and timezone: Czechia, UTC+1 Prague
Any specific Quarkus related content you’d be interested in hearing about during your session: no

**Reply by charger28** [reply]
@xstefank - just sent you an email to solidify dates/topics.

**Comment by lilian-benoit** [comment]
Hello Quarkus Team,

- **Name of the JUG** : [BordeauxJUG](http://www.bordeauxjug.org)
- **Best JUG Contact information**  : [Me](lilian-benoit@lbenoit.fr) or [Team](team@bordeauxjug.org)
- **Language preference** : French
- **Country and timezone** : France (UTC+1)
- **Any specific Quarkus related content you’d be interested in hearing about during your session** : No suggestion, but we are already 2 sessions quarkus : 
       In 2019, [Quarkus: Supersonique Subatomique Java ](https://www.meetup.com/BordeauxJUG/events/263399748/)
       In 2021, [Quarkus World Tour s'arrête à Bordeaux](https://www.meetup.com/BordeauxJUG/events/280673402/)

**Reply by charger28** [reply]
@lilian-benoit - just sent you an email to solidify date/topics

**Reply by charger28** [reply]
@lilian-benoit - can you reach out to me (jebeck@redhat.com) when you get a chance so we can solidify the dates and topics for your Quarkus World tour stop. Thanks!

**Comment by ehsavoie** [comment]
**Name of the JUG:** [AlpesJUG](https://www.alpesjug.org)
**Best JUG Contact information:** [Me](mailtp:administrateur@alpesjug.org) or [Team](mailto:jug-leaders@alpesjug.org)
**Language preference:** French
**Country and timezone;** Paris / Europz CET (UTC+1)
**Any specific Quarkus related content you’d be interested in hearing about during your session:** Nope, it has been a while since we had some Quarkus talk

**Comment by AnHaug** [comment]
Name of the JUG : [JUGM](https://www.jugm.de/)
Best JUG Contact information : ah@jugm.de
Language preference : German, English
Country and timezone : Germany (UTC+1)
Any specific Quarkus related content you’d be interested in hearing about during your session : Advanced stuff please, we already had several talks (Intro, GraalVM, Extensions)

**Reply by charger28** [reply]
@AnHaug - just sent you a private mail to solidify dates/topics.

**Comment by atomfrede** [comment]
- Name of the JUG: [Jug Paderborn (JUGPB)](https://jug-pb.gitlab.io/)
- Best Contact: [Twitter](https://twitter.com/jugpaderborn), [me](https://github.com/atomfrede/) or [Thorben](thjanssen)
- Language preference: German, English
- Country and timezone : Germany (UTC+1)
- Any specific Quarkus related content you’d be interested in hearing about during your session : We had no talk about quarkus before, so anything is fine.

**Reply by charger28** [reply]
@atomfrede - can you send me an email (jebeck@redhat.com) so we can solidify dates/topics?

**Reply by LightGuard** [reply]
@atomfrede when are you looking for a presentation? Also are you looking for in-person or virtual?

**Reply by atomfrede** [reply]
@LightGuard We would like to start in person meetups again. Some date in June would be nice (or July).

**Comment by mgrygles** [comment]
Name of the JUG:   [Chicago JUG](https://www.meetup.com/ChicagoJUG/)
Best JUG Contact information:  [Mary Grygleski](mailto:mary.grygleski@cjug.org)
Language preference:  English
Country and timezone: United States, US-Central Daylight Time (GMT-5)
Any specific Quarkus related content you’d be interested in hearing about during your session:  We are all open to any relevant topics. FYI we do have a good number of Spring users.

Thanks👍🏻

**Reply by charger28** [reply]
@mgrygles - I just sent you a personal email to solidify dates/topics.

**Comment by wilvdb** [comment]
Hello,

Name of the JUG : [YAJUG](https://yajug.lu/)
Best JUG Contact information : wilfried.vandenberghe@yajug.org or gildas.cuisinier@yajug.org
Language preference : French or English
Country and timezone: Luxembourg (UTC+2)
Any specific Quarkus related content you’d be interested in hearing about during your session : Any relevant topics would be good.

Thanks!

**Reply by LightGuard** [reply]
@wilvdb When would you be looking to have the presentation? Also, in-person or virtual?

**Reply by wilvdb** [reply]
@LightGuard  It would be nice to schedule the presentation in September or after. In-person meetup would be nice too.

**Reply by LightGuard** [reply]
Alright, I'll put you down for September, thanks!

**Reply by LightGuard** [reply]
@wilvdb would you be open to a virtual speaker if needed?

**Reply by wilvdb** [reply]
@LightGuard is it possible to schedule the event for the second part of September ? Everybody will be on holiday until mid of September. It's not possible for us to handle a virtual event.

**Reply by LightGuard** [reply]
I'll put that down, thank you!

**Reply by wilvdb** [reply]
@LightGuard do you have any news ? Will it possible to organize something in Luxemburg ?

**Comment by italocosta** [comment]
Hello,

Name of the JUG: CEJUG (https://github.com/cejug)
Best JUG Contact information:  [@ItaloCosta](mailto:italo.costa.ce@gmail.com)
Language preference: English or Portuguese
Country and timezone: Brazil (GMT-3)
Any specific Quarkus related content you’d be interested in hearing about during your session: It will be a pleasure to hear any Quarkus topic-related.

Thanks

**Reply by LightGuard** [reply]
@italocosta When would you like to have the presentation, and are you looking for in-person or virtual?

**Comment by conorhayes** [comment]
Hi Quarkus,

Name of the JUG: Cork JUG
Best JUG Contact information: [Conor](mailto:hayesc9807@gmail.com) [Mahsa](mailto:mnt32425@gmail.com)
Language preference: English
Country and timezone: Ireland UTC+1
Any specific Quarkus related content you’d be interested in hearing about during your session: No previous Quarkus talks, so we are open to anything.

Thanks!

**Reply by insectengine** [reply]
@conorhayes - I've sent an email to the provided address to discuss the details.

**Reply by conorhayes** [reply]
Thank you. Just replied there.

**Comment by erickinsella** [comment]
Hello Q,

Name of the JUG: North Star Dev https://twitter.com/NorthStarDevs
Best JUG Contact information:  @sabersd ([sabersd@gmail.com](mailto:sabersd@gmail.com)) and I @erickinsella ([erickinsella@gmail.com](mailto:erickinsella@gmail.com))
Language preference: English
Country and timezone: USA (GMT-5)
Any specific Quarkus related content you’d be interested in hearing about during your session: No, previous Quarkus talks but might be interesting to compare it to Micronaut and Helidon. The features that make it stand out about the rest.

**Comment by erilor** [comment]
Hi Quarkus!

Name of the JUG: Umea JUG
Best JUG Contact information: @attiand or @erilor or [Umejug@gmail.com](mailto:Umejug@gmail.com)
Language preference: English
Country and timezone: Sweden UTC+2
Any specific Quarkus related content you’d be interested in hearing about during your session: Any Quarkus-related talks, we haven't had any yet.

**Reply by insectengine** [reply]
@erilor  @attiand - Sent an email to discuss a tour stop in Umea.

**Comment by LightGuard** [comment]
July would probably be easier if you’re looking for in person. Which date
is your meeting?

**Reply by atomfrede** [reply]
July is also fine. We don't have a fixed date yet. But the 20th of July would be fine for us.

**Reply by LightGuard** [reply]
Great! Do you have topics you would like to be presented?

**Comment by rokon12** [comment]
Hello,

**Name of the JUG**: JUGBD (https://jugbd.org/about/)
**Best JUG Contact information**: @rokon12  or **bazlur@jugbd.org**
**Language preference**: English or Bangla 
**Country and timezone**: Bangladesh (GMT+6)
**Any specific Quarkus-related content you’d be interested in hearing about during your session**: No, any content would be good.

Thank you very much

**Reply by insectengine** [reply]
@rokon12  - Sent you an email to discuss a tour stop at JUGBD.

**Comment by swseighman** [comment]
Name of the JUG: Cleveland (Ohio) JUG [(https://www.meetup.com/cleveland-java/)](https://www.meetup.com/cleveland-java/)
Best JUG Contact information: Craig Castalez (craig@techelevator.com) or Scott Seighman (scott.seighman@gmail.com)
Language preference: English
Country and timezone: US/Eastern (GMT-5)
Any Quarkus-related content

**Reply by insectengine** [reply]
@swseighman  - Sent you an email to discuss a tour stop in Cleveland.

**Comment by dearrudam** [comment]
Hello, Quarkus Team!

**Name of the JUG:** SouJava : [Meetup](https://www.meetup.com/soujava/) - [Github](https://github.com/soujava)  
**Best JUG Contact information:** [Maximillian Arruda](mailto:dearrudam@gmail.com) (email: dearrudam@gmail.com |  Twitter: @maxdearruda)
**Language preference:** English or Portuguese
**Country and timezone:** Brazil (GMT-3)
**Any specific Quarkus related content you’d be interested in hearing about during your session:** It will be a pleasure to hear any Quarkus topic-related.

**Reply by insectengine** [reply]
@dearrudam - Emailed you regarding your request for a stop at SouJava.

**Reply by dearrudam** [reply]
@insectengine I've replied to this email yesterday.

**Comment by leeturner** [comment]
Hi Quarkus Team

**Name of the JUG:** Brighton JUG - [Meetup](https://www.meetup.com/brighton-jug/) - [Website](http://brightonjug.com/)
**Best JUG Contact information:** [Lee Turner](https://twitter.com/leeturner)
**Language preference:** English
**Country and timezone:** UK, BST

Any specific Quarkus related content you’d be interested in hearing about during your session:

Anything really.  Native binary cli applications, security, native microservices, how to deploy native to docker/k8s ?

**Reply by insectengine** [reply]
@leeturner - Can you send me an email so we can discuss a tour stop in Brighton? My email is jcobb@redhat.com.

**Reply by leeturner** [reply]
Sure thing.

**Comment by sherine-k** [comment]
Hi Quarkus team
We are the Marseille JUG
You can contact @sherine-k (sherinekhoury@gmail.com)

**Language preference** French 
**Country and timezone** France CEST
**Any specific Quarkus related content you’d be interested in hearing about during your session** we already are in contact with Quarkus contributors like @ia3andy and @FroMage , but don't know if their presentations of Renarde and Quinoa would qualify for the tour.

cc @jotak @vietj @tsegismont

**Reply by insectengine** [reply]
@sherine-k  - Sent you an email to discuss a tour stop in Marseille.

**Comment by iamsteveholmes** [comment]
So not officially a JUG but we sort of absorbed the local JUG during Covid.  

Name of the JUG
[Coder Consortium](https://www.meetup.com/Coder-Consortium/)

Best JUG Contact information
[Steven Holmes](mailto:iamsteveholmes@gmail.com)

Language preference
English

Country and timezone
Sacramento, California, United States, Pacific Standard Time

Any specific Quarkus related content you’d be interested in hearing about during your session
We often speak about alternative JVM languages, particularly Kotlin but also Scala and Groovy.  But in general we would love to hear about general features, boot times, memory footprint, etc.

**Reply by insectengine** [reply]
@iamsteveholmes - I sent you and email to discuss a tour stop in Sacramento.

**Comment by gquintana** [comment]
_Name of the JUG_
LyonJUG https://lyonjug.org/

_Best JUG Contact information_
team at lyonjug.org

_Language preference_
French    

_Country and timezone_
Lyon, France

_Any specific Quarkus related content you’d be interested in hearing about during your session_
We already had a Quarkus presentation a long time ago https://lyonjug.org/2019/09/30/quarkus-et-graal-vm.html
We needed a refresher or something to go deeper.

**Reply by insectengine** [reply]
@gquintana  - Sorry for the delay. I've sent an email to discuss.

**Comment by andotorg** [comment]
Hello,

Name of the JUG: JUG Andot (https://github.com/andotorg)
Best JUG Contact information: @andotorg ([andotorg@outlook.com](mailto:andotorg@outlook.com))
Language preference: English or Chinese
Country and timezone: China (GMT-8)
Any specific Quarkus related content you’d be interested in hearing about during your session: No, any content would be good.
Thank you very much!

**Reply by maxandersen** [reply]
@insectengine

**Reply by insectengine** [reply]
email sent!

**Reply by insectengine** [reply]
@andotorg  -   - I haven't heard back from you. Can you check your junkmail or email me @ jcobb@redhat.com

**Comment by christophe-f** [comment]
Name of the JUG: Toronto JUG 
Best JUG Contact information: [Jonathan/Mike/Christophe](mailto:organizers@tjug.ca)
Language preference: English 
Country and timezone: Toronto (EST)
Any specific Quarkus related content you’d be interested in hearing about during your session: Any Quarkus content would be excellent.

**Reply by insectengine** [reply]
email sent!

**Comment by frostcs7** [comment]
Name of the JUG: BERLIN JUG(http://www.jug-berlin-brandenburg.de/)
Best JUG Contact information: jindalsandeep47@gmail.com
Language preference: English
Country and timezone: CET
Any specific Quarkus-related content you’d be interested in hearing about during your session: Any Quarkus content would be excellent.

**Reply by insectengine** [reply]
@frostcs7  - Replied via email.

**Reply by insectengine** [reply]
@frostcs7  - I haven't heard back from you. Can you check your junkmail or email me @ jcobb@redhat.com

**Comment by ia3andy** [comment]
If needed, I could present Quarkus then do a demo of a full stack app with
Quarkus and Quinoa.

On Thu, Apr 6, 2023 at 4:47 PM insectengine ***@***.***>
wrote:

> @frostcs7 <https://github.com/frostcs7> - Replied via email.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarkusio/quarkus/discussions/23453#discussioncomment-5544531>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAQ664CS5XRLIU2QQN3445TW73JQJANCNFSM5NSX5LKA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Andy Damevin

**Comment by mariasalcedo** [comment]
Name of the JUG: JUG Ingolstadt (https://jug-in.bayern/)
Best JUG Contact information: @heubeck florian.heubeck@jug-in.bayern
Language preference: German and English
Country and timezone: Germany, Central European Time
Any specific Quarkus related content you’d be interested in hearing about during your session: Any Quarkus content would be excellent.

**Reply by insectengine** [reply]
@heubeck - Replied via email.

**Comment by magoeke** [comment]
A bit late to the game but I hope we can still make this work :)

Name of the JUG: [JUGS](https://www.jugs.org/)
Best JUG Contact information: jugs@magoeke.de
Language preference: German or English
Country and timezone: Germany, Central European Time
Any specific Quarkus related content you’d be interested in hearing about during your session: Anything would be amazing.
